Gemfile.lock

```diff
@@ -74,31 +73,11 @@ GEM
       coffee-script-source
       execjs
     coffee-script-source (1.12.2)
-    concerto_calendar (0.0.9)
```

Comment:
@zr2d2 - I'm surprised to see so many things removed from Gemfile.lock here.  This looks like a much larger change than the Nokogiri one.  Dependabot seems confused.